sdk/client/resourcemanager/poller_lro.go

```diff
@@ -203,6 +203,8 @@ func (p *longRunningOperationPoller) Poll(ctx context.Context) (result *pollers.
 			"Scaling": pollers.PollingStatusInProgress,
 			// HealthBot @ 2022-08-08 (HealthBots CreateOrUpdate) returns `Working` during Creation
 			"Working": pollers.PollingStatusInProgress,
+			// ServiceFabricManaged @ 2021-05-01 (NodeTypes CreateOrUpdate) returns `Created` rather than `InProgress` during Creation
+			"Created": pollers.PollingStatusInProgress,
```

Comment:
just to double-check, `Created` is used to mean `InProgress` rather than `Succeeded`?

Reply:
(minor, but could we also move this up, since the other keys are sorted alphabetically)